src/Stratis.Bitcoin/P2P/PeerConnectorAddNode.cs

```diff
@@ -41,6 +41,18 @@ public override void OnInitialize()
             }
         }
 
+        /// <summary>This connector is always started </summary>
+        internal override bool OnCanStartConnectAsync
```

Comment:
Making this method `internal` will prevent future implementation of a connector, why not use `protected`?

Reply:
You are right, but they should actually be public, not protected as the concrete implementation implements the behaviour.